cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -118,6 +118,14 @@ def _create_entrance_exam(request, course_key, entrance_exam_minimum_score_pct=N
         is_entrance_exam=True
     )
 
+    # Create the entrance exam section item.
+    create_xblock(
+        parent_locator=unicode(created_block.location),
+        user=request.user,
+        category='sequential',
+        display_name='Exam Sequential - Subsection'
```

Comment:
Sorry for the late response. Would this be the display name of the entire exam, or just one section of it? If it's the entire exam, I'd change the display name to just "Entrance Exam."

Also, I'm assuming course teams won't be able to change this name, correct?